modules/contact/include/problems/AugmentedLagrangianContactProblem.h

```diff
@@ -44,7 +44,15 @@ class AugmentedLagrangianContactProblem : public ReferenceResidualProblem
                             const Real ref_resid,
                             const Real div_threshold) override;
 
+  const unsigned int & getNumLagrangianUpdateIterations() const
```

Comment:
I'm assuming you're returning by reference for a reason?

Reply:
Yeah because I use it repeatedly